client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/index.jsx

```diff
@@ -25,6 +24,7 @@ import NotificationBar, {
   notificationShape,
 } from 'lib/components/NotificationBar';
 import withRouter from 'lib/components/withRouter';
+import palette from 'theme/palette';
```

Comment:
I revert the functional component for this file as there are a few regressions on the hooks. Also it's to demonstrate that it's possible to not use useTheme hook when working with class component by importing the palette directly